homeassistant/components/transport_nsw/sensor.py

```diff
@@ -70,6 +75,8 @@ class TransportNSWSensor(SensorEntity):
     """Implementation of an Transport NSW sensor."""
 
     _attr_attribution = "Data provided by Transport NSW"
+    _attr_device_class = SensorDeviceClass.DURATION
+    _attr_state_class = SensorStateClass.MEASUREMENT
```

Comment:
Please split this change of into a separate PR

Reply:
I have removed these as requested but can you explain why this enhancement which relates to the bug raised should be in a seperate PR?

Reply:
The PR title is "Resolve bugs with Transport NSW", which is related to the library returning n/a.
Setting the device class and state class is a welcome change, but it should be done in a separate PR because it's not related to the bug fix.